packages/xchain-polkadot/src/client.ts

```diff
@@ -31,74 +31,154 @@ export interface PolkadotClient {
   estimateFees(params: TxParams): Promise<Fees>
 }
 
+/**
+ * Custom Polkadot client
+ */
 class Client implements PolkadotClient, XChainClient {
   private network: Network
   private phrase = ''
   private address: Address = ''
   private api: ApiPromise | null = null
   private derivationPath = "44//354//0//0//0'"
 
+  /**
+   * Constructor
+   * Client is initialised with network type
```

Comment:
`Client is initialised with network type` -> `Client is initialised with network type and phrase (optional)`